### DISCUSSION TITLE
FreeRDP-WebConnect portable in a ZIP file?

### DISCUSSION THREAD

**Discussion by mkanet** [FreeRDP/FreeRDP#10146] [Q&A]
I wasn't able to find a portable version of **FreeRDP-WebConnect** Is it possible for me to install it on a PC I dont care about; and, just copy the program directory and respective files to a new PC; and, expect it to work?  I would much more prefer using a portable installation.

**Comment by akallabeth** [comment]
1. wrong repository
2. https://github.com/FreeRDP/FreeRDP-WebConnect is archived
